stader-cli/node/status.go

```diff
@@ -217,26 +216,21 @@ func getNodeStatus(c *cli.Context) error {
 		log.ColorReset,
 		(eth.WeiToEth(maxUtilizable)))
 
-	fmt.Printf(
-		"The SD utility can provide max of %.6f SD.\n\n",
-		math.RoundDown(eth.WeiToEth(sdStatus.PoolAvailableSDBalance), eth.Decimal))
-
 	current := eth.WeiToEth(totalCollateral)
 	require := eth.WeiToEth(sdStatus.SdCollateralRequireAmount)
 
 	fmt.Printf(
-		"The node %s%s%s current had %.6f%s Collateral.\n\n",
+		"The node %s%s%s current had %.6f%s Collateral.\nPlease ensure that the SD collateral percentage is greater than %s. The SD collateral snapshots are taken daily at a random block, and if the SD collateral value falls below the %s limit, the node operator will not earn SD rewards for that day\n\n",
 		log.ColorBlue,
 		status.AccountAddress,
 		log.ColorReset,
-		current/require*100, "%")
+		current/require*10, "%", "10%", "10%")
```

Comment:
we should multiply by 100 right?

Reply:
require is 10%, so we should mul by 10 only bro